frontend/vite.config.ts

```diff
@@ -1,58 +1,45 @@
 import { readFileSync } from 'node:fs'
-import { resolve } from 'node:path'
 import { execSync } from 'node:child_process'
-import { defineConfig, loadEnv } from 'vite'
+import { defineConfig } from 'vite'
 import solidPlugin from 'vite-plugin-solid'
 import unocssPlugin from 'unocss/vite'
 
 function eruda() {
-  const erudaSrc = readFileSync("./node_modules/eruda/eruda.js", "utf-8");
+  const erudaSrc = readFileSync('./node_modules/eruda/eruda.js', 'utf-8')
   return {
-    name: "vite-plugin-eruda",
-    apply: "build",
-    transformIndexHtml(html) {
+    name: 'vite-plugin-eruda',
+    transformIndexHtml(html: any) {
       const tags = [
         {
-          tag: "script",
+          tag: 'script',
```

Comment:
would be better to not mix formatting changes next time.

Reply:
okay :+1: